Makefile

```diff
@@ -193,6 +193,7 @@ csv-clean: ## Remove existing OLM files.
 crds: $(CONTROLLER_GEN) $(YQ)
 	@echo Updating CRD manifests
 	@build/crds/build-crds.sh $(CONTROLLER_GEN) $(YQ)
+	@build/crds/generate-crd-docs.sh
```

Comment:
```suggestion
	@GOBIN=$(GOBIN) build/crds/generate-crd-docs.sh
```

Reply:
Is it needed because we are installing a go package?

Reply:
If yes I think the `generate-docs-crds` wouldnt work for you